sites/public/src/components/account/ConfirmationModal.tsx

```diff
@@ -5,6 +5,7 @@ import { useRouter } from "next/router"
 import { useContext, useEffect, useRef, useState } from "react"
 import { useForm } from "react-hook-form"
 import { emailRegex } from "../../lib/helpers"
+import { ParsedUrlQuery } from "querystring"
```

Comment:
Where is this package pulled in? This type is also just `{ [key: string]: string | string[]; }` - as the type is fairly generic, I don't think we need to pull the package in.